src/components/content-list/content-list.tsx

```diff
@@ -45,9 +45,9 @@ export async function ContentList(props: WidgetContext<ContentListEntity>) {
         if (properties.SelectedItems && properties.SelectedItems.Content && properties.SelectedItems.Content.length > 0) {
             const selectedContent = properties.SelectedItems.Content[0];
             const detailModel = await handleDetailView({
-                Id: properties.SelectedItems.ItemIdsOrdered[0],
+                Id: properties.SelectedItems.ItemIdsOrdered![0],

```

Comment:
TBO not sure - I changed the type because I stumbled upon a single case where this was possibly null, this forced me to change the other ones with '!' because they were initially working without the null assumption. I will add null checks to avoid any possible errors.